falcon-eye-org/client-js
# Feature/handshake

## PR: feature/handshake > dev

### What's new ?

- Added a handshake when the client has no profileId (can't be recognized by the server)

### How to test it ?

- You need to first build the script (`npm run build`) then start the server (`npm run server`).  When it's done you have to connect to the server with your browser then refresh the webpage and check your browser's localStorage (just type localStorage in the browser console).
